x-pack/plugins/apm/server/index.ts

```diff
@@ -53,6 +53,9 @@ const configSchema = schema.object({
     onboarding: schema.string({ defaultValue: 'apm-*' }),
   }),
   forceSyntheticSource: schema.boolean({ defaultValue: false }),
+  latestAgentVersionsFileUrl: schema.string({
```

Comment:
I think we can omit `File`
```suggestion
  latestAgentVersionsUrl: schema.string({
```

Reply:
Addressed in [f412ee2](https://github.com/elastic/kibana/pull/153643/commits/f412ee2e751f8f22655c749bec87159640b21972).